apps/postgresql-server/Dockerfile

```diff
@@ -10,24 +10,44 @@ RUN \
         /opt/postgresql-server/bin/ \
         /opt/postgresql-server/schema/ \
     && \
+    apt-get -y --no-install-recommends install python3 python3-pip python3-setuptools && \
+    #
+    # Upgrade Pip
+    pip3 install -U pip && \
+    #
+    # https://github.com/pypa/pip/issues/5221#issuecomment-382069604
+    hash -r pip3 && \
+    apt-get -y --no-install-recommends install build-essential python3-dev libffi-dev libpq-dev && \
+    #
+    # Install package to manage schema migrations
+    pip3 install yandex-pgmigrate==1.0.6 && \
+    #
+    # Remove temporary dependencies
+    apt-get -y remove build-essential python3-dev libffi-dev libpq-dev && \
+    #
+    # Cleanup
+    apt-get -y autoremove && \
```

Comment:
Add an extra:

```shell
apt-get -y clean
```

here too if you can. It [removes downloaded APT packages](https://www.networkworld.com/article/3453032/cleaning-up-with-apt-get.html).